### DISCUSSION TITLE
Runtime error after updating Avalonia version

### DISCUSSION THREAD

**Discussion by reelthymeos** [AvaloniaUI/Avalonia#17853] [Q&A] [answered]
Hi,

I have an app that was using Avalonia version 11.0.2. Other packages such as reactiveUI were at that version as well. I updated all packages via the nuget manager to the latest and it compiles fine, but when I attempt to run in debug mode I get the following exception:

> System.TypeLoadException
>   HResult=0x80131522
>   Message=Method 'CreateEmbeddableTopLevel' in type 'Avalonia.Win32.Win32Platform' from assembly 'Avalonia.Win32, Version=11.0.2.0, Culture=neutral, PublicKeyToken=c8d484a7012f9a8b' does not have an implementation.
>   Source=Avalonia.Win32
>   StackTrace:
>    at Avalonia.Win32ApplicationExtensions.<>c.<UseWin32>b__0_0()
>    at Avalonia.AppBuilder.SetupUnsafe()
>    at Avalonia.AppBuilder.Setup()
>    at Avalonia.AppBuilder.SetupWithLifetime(IApplicationLifetime lifetime)
>    at Avalonia.ClassicDesktopStyleApplicationLifetimeExtensions.StartWithClassicDesktopLifetime(AppBuilder builder, String[] args, Action`1 lifetimeBuilder)
>    at RcmxGui.Desktop.Program.Main(String[] args) in C:\Users\fooMan\source\repos\Foo.Desktop\Program.cs:line 14

I notice a reference to the original version 11.0.2 in the exception. Does something need to updated somewhere else as well?

**Comment by rabbitism** [comment, marked as answer]
Looks like you have two projects(core and desktop) and one of them is not updated

**Reply by Hunv** [reply]
Ouch... so obvious... Thank you for the hint 🤦‍♂️
